### DISCUSSION TITLE
Katana: create audio clips in your browser

### DISCUSSION THREAD

**Discussion by callmephilip** [katspaugh/wavesurfer.js#3689] [Show and tell]
Happy user of wavesurfer in [Katana](https://github.com/callmephilip/katana). Thank you for this great library

**Comment by katspaugh** [comment]
Nice work, thanks for sharing!
I'll be referring people to your project when a question about cutting/combining audio pops up (it happens quite often!) 🙂

**Reply by callmephilip** [reply]
sweet. thanks!
